### PR TITLE
Content and accessibility Sweep

### PR DESCRIPTION
**Pull request by autotelik**
https://eaflood.atlassian.net/browse/RIP-235

Resolving multiple small issues with content and css

### CODE REVIEW

File: app/views/flood_risk_engine/enrollments/steps/_check_exemptions.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<table class="check-your-answers"
-      summary="<%= t('flood_risk_engine.enrollments.steps.check_exemptions.table_summary') %>">
+<table class="check-your-answers">
+  <caption class="visuallyhidden">Exemption and link to change it</caption>
```
Comment (reggieb) [inline app/views/flood_risk_engine/enrollments/steps/_check_exemptions.html.erb]: Should the text "Exemption and link to change it" be in a translation file?

File: app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb [hunk 2]
```diff
@@ -1,35 +1,46 @@
...
-<p class="text">
-  <%= link_to t('.feedback_text'), t('.feedback_url'), class: 'link' %> <%= t('.feedback_helper') %>
-</p>
+    <h2 class="heading-medium"><%= t('.heading_h2_1') %></h2>
+
+    <p class="text"><%= t('.para_text7') %></p>
+    <p class="text"><%= t('.para_text8') %></p>
+
+    <p class="text">
+      <%= link_to t('.feedback_text'), t('.feedback_url'), class: 'link' %> <%= t('.feedback_helper') %>
```
Comment (reggieb) [inline app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb]: Is there not a better place to put URLs than the i18n translation files? Why not just specify the URLs within the HTML as that is normal practice.

File: app/views/flood_risk_engine/enrollments/steps/_grid_reference.html.erb [hunk 3]
```diff
@@ -4,75 +4,72 @@
...
-      <p><%= t(".details.tool_info")  %></p>
+      <ul class="list list-number">
+        <li>
+          <%= t(".details.bullet1") %>
+        </li>
+        <li>
+          <%= t(".details.bullet2") %>
+          <br/>
+          <%= t(".details.bullet3") %>
+        </li>
```
Comment (reggieb) [inline app/views/flood_risk_engine/enrollments/steps/_grid_reference.html.erb]: `bullet2a` and `bullet2b` may be clear - as both current 2 and 3 are part of the same "bullet" `<li>` item.

File: app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb (hunk 2 above)
Comment (autotelik) [inline app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb]: changed to use route page_path

Comment (woodpigeon) [inline app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb]: Agree urls should not be in translations. As this is an internal engine one I think I'd leave it in the html as if it was a normal url ie page_path(:feedback). 
I'd say external urls should be in config so they can be configured if necessary

**Comment by reggieb** [comment]
:ship: it

**Comment by woodpigeon** [comment]
Great stuff :ship: it from me

File: app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb (hunk 2 above)
Comment (autotelik) [inline app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb]: @woodpigeon  github diff bit weird : code was already changed to  

`<%= link_to t('.feedback_text'), page_path('feedback')  ...`

**Pull request merged**